hacks_mbof/settings.py

```diff
@@ -171,16 +171,24 @@
         'simple': {
             'format': '%(levelname)s %(name)s %(message)s'
         },
+        'ocellus_formatter': {
+            'format': '%(levelname)s %(asctime)s %(module)s %(message)s',
```

Comment:
I prefer the timestamp to be first on the line, but if you think this works better for some reason, it's OK.